pf4j/src/test/java/ro/fortsoft/pf4j/DefaultExtensionFactoryTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2015 Decebal Suiu
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ro.fortsoft.pf4j;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import ro.fortsoft.pf4j.plugin.TestExtension;
+
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Mario Franco
+ */
+public class DefaultExtensionFactoryTest {
+
+    public DefaultExtensionFactoryTest() {
```

Comment:
I think that we can remove empty constructor and empty `setUp`, `tearDown` methods to strip the code. Is it ok for you?